packages/client/components/ScopePhaseArea.tsx

```diff
@@ -136,10 +139,17 @@ const ScopePhaseArea = (props: Props) => {
         <TabContents>
           <ScopePhaseAreaParabolScoping isActive={activeIdx === 2} meeting={meeting} />
         </TabContents>
+        <TabContents>
```

Comment:
-1 do we need a feature flag here? Otherwise, we'll release this to the world when we ship